docs/realm-configuration.md

```diff
@@ -121,11 +121,11 @@ These configuration should be set for each UserStorage contained in a Realm :
 Those are optional properties to set on a userstorage. If the property is not set at the userstorage level, 
 the corresponding realm property will be used as default if set. 
 
-| Key                                                                          |                                                                                                             Description                                                                                                              |
-| ---------------------------------------------------------------------------- | :----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------: |
-| {name}\_send_login_template with name being a configured external webservice |   A template to complete and send to the {name} webservice on /send-login call (see [Notify external webservices](concepts.md#notify-external-webservices) and [Webhooks configuration](configuration.md#webhooks-configuration))    |
-| {name}\_reset_template with name being a configured webservice               | A template to complete and send to the {name} webservice on /reinit-password call (see [Notify external webservices](concepts.md#notify-external-webservices) and [Webhooks configuration](configuration.md#webhooks-configuration)) |
-| {name}\_changepwd_template with name being a configured webservice           | A template to complete and send to the {name} webservice on /change-password call (see [Notify external webservices](concepts.md#notify-external-webservices) and [Webhooks configuration](configuration.md#webhooks-configuration)) |
+| Key                                                                          |                                                                                                             Description                                                                                                              | Example |
+| ---------------------------------------------------------------------------- | :----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------: | -------- |
+| send_login_template |  A template to complete and send to a webservice on /send-login call (see [Notify external webservices](concepts.md#notify-external-webservices) and [Webhooks configuration](configuration.md#webhooks-configuration)). Should be prefixed by {name of the webservice}: . Several configuration can be set by separating them with a \| | spoc:https://spoc.insee.fr/template|test:http://test.insee.fr |
```

Comment:
not clear that the value expected is an URI to a template
(warn for a future issue  : it seems that if uri is not well formed, just an exeption is thrown and default template is not used)